src/OrchardCore.Modules/Orchard.Lucene/Startup.cs

```diff
@@ -42,6 +53,23 @@ public override void Configure(IApplicationBuilder app, IRouteBuilder routes, IS
                 template: "Search/{id?}",
                 defaults: new { controller = "Search", action = "Index", id = "" }
             );
+
+            routes.MapAreaRoute(
+                name: "Api.Lucene.Content",
+                areaName: "Orchard.Lucene",
+                template: "api/lucene/content",
+                defaults: new { controller = "Api", action = "Content" }
+            );
+
+            routes.MapAreaRoute(
+                name: "Api.Lucene.Documents",
+                areaName: "Orchard.Lucene",
+                template: "api/lucene/documents",
```

Comment:
Yes, I made a permission for it too. Disabled on anonymous by default. I think that we will extend it later to allow more granular permission, i.e. per query.